openshift/configmap-template.yaml

```diff
@@ -46,8 +46,8 @@ objects:
       prometheus-pushgateway-host: ${PROMETHEUS_PUSHGATEWAY_HOST}
       prometheus-pushgateway-port: ${PROMETHEUS_PUSHGATEWAY_PORT}
       metrics-pushgateway-url: ${PROMETHEUS_PUSHGATEWAY_HOST}:${PROMETHEUS_PUSHGATEWAY_PORT}
-      janusgraph-service-host: ${JANUSGRAPH_SERVICE_HOST}
-      janusgraph-service-port: ${JANUSGRAPH_SERVICE_PORT}
+      dgraph-service-host: ${DGRAPH_SERVICE_HOST}
+      dgraph-service-port: ${DGRAPH_SERVICE_PORT}
```

Comment:
Let's stick with the current naming not to introduce confusion:

- `GRAPH_SERVICE_HOST`
- `graph-service-host`

 https://github.com/thoth-station/storages/blob/f2ae62f8fc040106b851b0d0ff759f1ef686ee5c/thoth/storages/graph/dgraph.py#L123